packages/flutter/lib/src/material/button_theme.dart

```diff
@@ -120,10 +120,6 @@ class ButtonTheme extends InheritedTheme {
   }) : assert(data != null),
        super(key: key, child: child);
 
-  // TODO(darrenaustin): remove after this deprecation warning has been on
```

Comment:
I removed this sine the 'Deprecated' below will be culled later on during deprecation reaping. No need for a todo.